samples/ipc_call_interface/consumers/my_consumer.c

```diff
@@ -4,13 +4,21 @@
 
 #include "my_consumer.h"
 #include "az_ulib_result.h"
+#include "azure/az_core.h"
 #include "wrappers/cipher_1_wrapper.h"
 #include <inttypes.h>
 #include <stdint.h>
 #include <stdio.h>
 
 static az_ulib_ipc_interface_handle _cipher_1;
-#define BUFFER_SIZE 100
+#define BUFFER_SIZE 200
```

Comment:
can you add a description for how you landed on this buffer size so we know how to tweak it if need be?